build.sh

```diff
@@ -1,28 +0,0 @@
-#!/bin/bash
```

Comment:
Would keeping build.sh be useful? When you're developing locally you can quickly create a zip to test it out instead of having to commit + push the code already.